sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/_models.py

```diff
@@ -32,6 +39,25 @@ def __init__(
         self.format_version = kwargs.get("format_version", None)
         self.storage_source = kwargs.get("storage_source", None)
 
+    def to_generated(self):
```

Comment:
I know this is misleading because the linter checks will tell you _not_ to access a method prefixed with underscore, but we don't want our users to access these model methods. So we can keep them "private" and leave the underscore prefix, but add a linter ignore comment to any of the lines that call them: `# pylint: disable=protected-access`